source/includes/imports/list.md

```diff
@@ -0,0 +1,57 @@
+# List of metrics reported cluster wide
```

Comment:
Reviewers can safely ignore this file.

Note: I wasn't able to find a good way to simply insert this file into existing restructured text. That would have allowed offloading the maintenance of the metrics list onto the main minio server. 

The alternative would be to use either restructured text for this file in the MinIO server docs, *or* to use the `Myst` syntax - which I need to experiment with. If that syntax works, there might be more offloading we can do to retain lists like this in the server code for easier maintenance over time.